tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -186,5 +187,22 @@ public void TestMethod(string str)
         if(TestString != str)
             throw new Exception();
     }
+    public virtual void TestVirtualMethod(string str)
+    {
+        Program.PrintLine("Virtual Slot Test: Ok If second");
+    }
+}
+
+public class TestDerivedClass : TestClass
+{
+    public TestDerivedClass(int number) : base(number)
+    {
+
+    }
+    public override void TestVirtualMethod(string str)
```

Comment:
Can you please add another virtual method so that we can ensure slots are dispatching to the right method?